wikimedia/wikipedia-ios
# Apply user's preferred language variant to incoming links when applicable

**Phabricator:** https://phabricator.wikimedia.org/T278581

### Notes
* Applies user's preferred language variant code to incoming URLs if they match the site URL of the incoming link.

### Test Steps
When tapping a top read widget row element, it should open in the app in your primary (preferred) language/variant.

1. Add a language with variants (Chinese, Serbian, etc.) as your primary app language
2. Add Top read widget to home screen
3. Tap a top read element on widget, confirm the app opens to that article in your current primary language variant
4. Update your variant to a different variant in the same language
5. Repeat step 3
